.prettierrc

```diff
@@ -5,5 +5,6 @@
   "semi": true,
   "trailingComma": "all",
   "arrowParens": "always",
-  "tabWidth": 2
+  "tabWidth": 2,
+  "endOfLine": "auto"
```

Comment:
Why "auto"? Default is "lf" which means lines always end in \n instead of \r\n.

Reply:
Its prettier on my Windows 11 VS Code that doesn't like endofline="lf", where "auto" fixed the problem for me, however Ill try to force Git and VS Code to use '\n' only from now on. 

Check if the Name.tsx file is ok with your system regarding the endofline characters.